src/get-config.js

```diff
@@ -38,3 +38,15 @@ export function getBiliConfig(_config) {
   }
   return config
 }
+
+export function getPrettierConfig() {
+  const useConfig = new UseConfig({
+    name: 'prettier',
+    files: ['.{name}rc', '{name}.config.js', '.{name}rc.js', 'package.json']
+  })
+  const { path: configPath, config } = useConfig.loadSync()
+  if (configPath === undefined) {
+    throw new BiliError('Cannot find Prettier config file.')
+  }
+  return config
+}
```

Comment:
I think prettier itself has a method for resolving config file: https://prettier.io/docs/en/api.html#prettierresolveconfigfilepath-options

Reply:
lemme take a loop

Reply:
So you are fine with adding Prettier as a straight dependency to project only to use this method?

Reply:
both `prettier` and `rollup-plugin-prettier` can be `this.localRequire`-ed I think.

Reply:
honestly I think the prettier config file should be automatically loaded by that rollup plugin 😅

Reply:
Naaah, does it?!

Reply:
it doesn't, we should send a PR to add this feature

Reply:
So just tell me what are your final thoughts on this one,
I load Prettier with `localRequire` and use it's `resolveConfig` method?
Or it's good the way it is now?
or gonna send PR?  :sweat_smile:

Reply:
ok just localRequire for now.

Reply:
If I `localRequire` prettier, then user should have had prettier installed, while it wouldn't be true version of prettier which is in use ( user probably will be expecting it to be), the true version would be prettier of `rollup-plugin-prettier`, so user is installing prettier only for us to use it's `resolveConfig` method. Am I right?

> both `prettier` and `rollup-plugin-prettier` can be `this.localRequire`-ed I think.

And if we force user to install both of them it isn't really user friendly imo.
What do you think? I will go however you want it.

Reply:
OK I looked at those plugin, most Rollup and Webpack plugins does not support external conf files, 90%  get configs where those are called, so even if PR it might never merge.

Implementing a `getWhateverConfig` for every plugin sounds silly (like `getPrettierConfig`) and most plugins do not support conf files, so I came up with an idea that we could create a general function that user can load any conf file he wants inside bilirc, like:
```js
prettier: {
  getConfig: './.prettierrc'
}
```
or for anything else since now there is no support for terser conf file either:
```js
terser: {
    getConfig: './config/terser.json'
}
``` 
So that whenever you wanted to add a plugin you don't have worry whether it support conf file or not, and also users will get a consistent syntax.

Or

If you don't want it this way for any reason, we can just ignore all conf files (if the plugin does not support) and force user to put all of their setting inside bilirc :
```js
prettier: {
  space: 2,
  emptyLine: true
  ...
}
```
Which?

Reply:
if the library used by the rollup plugin supports config files natively, then the rollup plugin should support it, like rollup-plugin-babel

Reply:
if the library does not support config file itself, like terser, then there's no point adding config file support for it in bili, since the file is not used by any other tool. 😂

Reply:
kay, so I will fix my PR, and then if have time I'll try to fix the plugin too :)